lib/wallet/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -2440,6 +2442,9 @@ instance Arbitrary (Hash "BlockHeader") where
 instance Arbitrary ApiBlockHeader where
     arbitrary = genericArbitrary
     shrink = genericShrink
+
+instance Arbitrary TxId where
+    arbitrary = TxId <$> arbitrary
```

Comment:
Please put the `Arbitrary` instance in the `StoreSpec` module — it's ok and even desired for `Arbitrary` instances to be duplicated, see [Coding Standards — Test files are separated and self-contained](https://input-output-hk.github.io/adrestia/code/Coding-Standards#test-files-are-separated-and-self-contained): "Arbitrary instances are not shared across test files and are defined locally.".

Also, the main concern of the `Api.TypeSpec` module is testing HTTP API; the type `TxId` from the database schema does not belong here.